### ISSUE TITLE
Feature Request: Webview (and Custom chrome tabs for Android)

### ISSUE THREAD

**Issue by snazzy92** [kivy/plyer#627]
Is it possible to create an API to call Webviews and custom Chrome tabs from Plyer? Pyjnius is just not very intuitive to work with.

**Comment by RobertFlatt** [comment]
It is possible to instantiate a Webview without using Plyer, for example:
https://github.com/RobertFlatt/Android-for-Python/tree/main/webview
Android only.
I know nothing about Chrome tabs.

**Comment by snazzy92** [comment]
Thanks for that! I've actually been using this Webview for my application.
I still think we need an easier and platform independent way to call webviews for Kivy apps. I know webviews exist in python and you can call them from other libraries like pyqt but those weren't made with android/ios in mind and may not work on them.
Custom chrome tabs:
https://developer.chrome.com/docs/android/custom-tabs/overview/
Basically, its an improved version of Webviews that gives you faster load times through pre-fetching and control over which browser you want to use in your app.
